internal/clients/manager.go

```diff
@@ -47,6 +52,12 @@ type AdminAPIClientsManager struct {
 	// endpoints list that should be used for configuring the dataplane.
 	discoveredAdminAPIsNotifyChan    chan []adminapi.DiscoveredAdminAPI
 	gatewayClientsChangesSubscribers []chan struct{}
+	// adminAPIServiceName is the name of admin API service.
```

Comment:
```suggestion
	
	// adminAPIServiceName is the name of admin API service.
```

Reply:
removed this because it is not required.